src/main/java/Controllers/ExceptionController.java

```diff
@@ -0,0 +1,25 @@
+package Controllers;
+
+import Exceptions.IndexOutOfBounds;
+import Exceptions.RecordNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.javapoet.ClassName;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class ExceptionController {
+
+
+    @ExceptionHandler(value = RecordNotFoundException.class)
+    public ResponseEntity <Object> exception (RecordNotFoundException exception){
+        return new ResponseEntity<>(exception.getMessage(), HttpStatus.NOT_FOUND);
+    //exception.getMessage() kan ook "dit is een tekst"
```

Comment:
In plaats van `exception.getMessage()` mag je inderdaad ook een hardcoded String returnen, zoals "dit is een tekst". Wat je hier invult is de tekst die de gebruiker in de frontend of postman te zien krijgt.